### DISCUSSION TITLE
How can I adjust the width of the completionItem popup to display all the text?

### DISCUSSION THREAD

**Discussion by jiangtao69039** [TypeFox/monaco-languageclient#705] [Q&A] [answered]
The detail is too long to display all the content. Is it possible to adjust the width of the popup?
![image](https://github.com/TypeFox/monaco-languageclient/assets/20572131/30d1e344-fbc6-4811-aa39-2c3ee680d250)

**Comment by CGNonofr** [comment, marked as answer]
The short answer is: if it's possible with VSCode/monaco, yes, otherwise no

**Reply by jiangtao69039** [reply]
Thanks !
